src/machine.rs

```diff
@@ -91,8 +91,8 @@ pub struct Evaluator<'tcx> {
     pub(crate) argv: Option<Pointer<Tag>>,
     pub(crate) cmd_line: Option<Pointer<Tag>>,
 
-    /// Last OS error.
-    pub(crate) last_error: Option<Pointer<Tag>>,
+    /// Last OS error location in memory. It is a 32 bits integer (unsigned for Windows)
```

Comment:
```suggestion
    /// Last OS error location in memory. It is a 32 bit integer (unsigned for Windows)
```
Right now it's always `u32`, isn't it?

Reply:
not in unix, for unix is an `i32`

Reply:
But you are always allocating a `u32` and accessing it as such.

However, the two are the same unless you do arithmetic.

Reply:
yes that's true. You think the distinction is pointless in the docs then?

Reply:
Yeah... better just say "32bit integer".